ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_match.txt

```diff
@@ -1,4 +1,4 @@
-r, "\.ERR", "\.WARN", "crash"
```

Comment:
I think this is a high impacting change. Can you elaborate the justification?

Reply:
From last 2 month, I didn't see a valid match with WARN, and actually all warning messages we collect, the decision we made is to move to white list, I think we don't need this for common match, if some case we really care the warnings, we can use the expect.